lib/swagger/paths.js

```diff
@@ -180,7 +194,10 @@ swaggerPaths._getPathOperationObject = function(options) {
   if (((options.handler === "search") || (options.handler === "find")) && !options.relation) {
     pathDefinition.parameters = pathDefinition.parameters.concat(swaggerPaths._optionalJsonApiParameters());
     responseShortcut.included = {
-      type: "array"
+      type: "array",
+      items: {
+        type: "object"
+      }
```

Comment:
This is the definition for what should appear in the `included` block on any request supporting inclusions. I've set it to simply be `an object` for now - it seems like overkill to say `this could be any resource in the swagger doc`.